packages/xy-chart/src/utils/computeCirclePack.js

```diff
@@ -1,7 +1,271 @@
-export default function computeCirclePack(data) {
+/* eslint no-param-reassign: 0 */
+/**
+  * Assume that the position of x and the size of circles have been scaled.
+  * And each circle has the following properties:
+  *  x: the position of x,
+  *  y: the position of y (can be undefined but will be calculated),
+  *  size: the radius,
+  *  importance: ,
+  */
+
+function packCircles(data, xScale) {
```

Comment:
I think it'd be awesome to add a few more comments describing how the algorithm works if not too much trouble. Also could be cool to mention this in the readme?

Reply:
Added the paper describing the algorithm.